person_signup.md

```diff
@@ -23,7 +23,7 @@ Some initial implementations may only support helpers -- direct RESTful access m
 * [Endpoints and URL structures](#endpoints-and-url-structures)
 * [Fields](#fields)
     * [Person Signup Helper Fields](#person-signup-helper-fields)
-    * [Person Signup Helper Answer Fields](#person-signup-helper-answer-fields)
+    * [Helper Action Functions](#helper-action-functions)
```

Comment:
Yeah I would call these Helper Action Fields, perhaps.  I guess I see the point of "functions" a bit more clearly here but I still prefer fields, as an aesthetic matter.